packages/cli/src/config/schema.ts

```diff
@@ -900,9 +893,9 @@ export const schema = {
 		},
 	},
 
-	binaryDataManager: {
+	binaryDataService: {
 		availableModes: {
```

Comment:
Renaming this is a breaking change, and should be documented in `BREAKING-CHANGES.md`

Reply:
if we are reverting, can we revert all of the renaming, and not just the config? either that, or we don't revert and list this as a breaking change.

Reply:
I think there is benefit in a temporary middle ground. Preexisting code uses "manager" for both binary data manager and its modes, so it's rather confusing and inconsistent. With this division into service and managers, the distinction is clearer, and the inconsistency is reduced to only this handful of entrypoint config spots where the renaming would cause a breaking change. So on balance the bulk of code becomes clearer.